test/common/index.js

```diff
@@ -37,10 +37,7 @@ const bits = ['arm64', 'mips', 'mipsel', 'ppc64', 'riscv64', 's390x', 'x64']
   .includes(process.arch) ? 64 : 32;
 const hasIntl = !!process.config.variables.v8_enable_i18n_support;
 
-const {
-  atob,
-  btoa
-} = require('buffer');
+const parseEslintConfigForGlobals = require('./parseEslintConfigForGlobals');
```

Comment:
Does this need to be a separate file? Should it be inlined so that people don't think it's something they can/should include themselves in their own tests?